app/controllers/movie_actors_controller.rb

```diff
@@ -0,0 +1,9 @@
+class MovieActorsController < ApplicationController
+  def create
+    @actor = Actor.find(params[:actor])
+    @movie = Movie.find(params[:movie_id])
+    ActorMovie.create!(actor: @actor, movie: @movie)
+
+    redirect_to "/movies/#{@movie.id}"
+  end
```

Comment:
Perfect. Great use of the join controller to make the connection between an actor and a movie!